src/ProcessResolver.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_PROCESS_RESOLVER_H_
+#define MUMBLE_PROCESS_RESOLVER_H_
+
+#include <stdint.h>
+#include <QtCore/QVector>
+
+/// This ProcessResolver can be used to get a QVector of running process names and associated PIDs on multiple platforms.
+/// This object is by no means thread-safe!
+class ProcessResolver {
+	protected:
+		/// The vector for the pointers to the process names
+		QVector<const char*> m_processNames;
```

Comment:
```suggestion
		QVector<const char *> m_processNames;
```

Reply:
Shouldn't `clang-format` have fixed this?

Reply:
Yes - but I'll run clang-format post merge. Haven't run it on this PR yet